okio/src/test/java/okio/BufferTest.java

```diff
@@ -582,4 +583,14 @@
     Buffer buf = new Buffer().write(new byte[] { 0, 1, 2, 3 });
     assertEquals(1, buf.snapshot(1).size());
   }
+
+  @Test public void readByteArrayThrowsEofException() throws Exception {
+    Buffer buf = new Buffer().write(new byte[] { 42 });
```

Comment:
supernit: `buf` –> `buffer`

It makes the code a little more boring if things have fewer names. Trying to guess what short forms are in use is more work than I want to do!